tests/lit-tests/explicit-vector.ispc

```diff
@@ -0,0 +1,73 @@
+// This test checks for the simplification of induction variables for loops and efficient
+// generation of accumulator widening/arithmetic.
+// See [GitHub issue ISPC/#2460](https://github.com/ispc/ispc/issues/2460) for details.
+
+// RUN: %{ispc} -DVECTOR=0 %s -o %t.S -O2 --nostdlib --emit-asm --target=avx2-i32x4 --arch=x86-64 --x86-asm-syntax=intel
```

Comment:
Use pipes where possible, it's more efficient and more compact/readable.

Reply:
I think it's less readable because I am not sure that lit tests support escaped line breaks, but it should save a write to disk.

Reply:
Typically we have multiple `RUN` commands for a lit test. If `FileCheck` is on separate line, then you need to match to what `ispc` invocation it corresponds to and make sure that it reads appropriate file. And after that start reading the command line for `ispc` to understand what's going on. With multiple `RUN` commands it's just a prerequisite step for understanding what's going on in the test - i.e. it's unnecessary distraction.

`FileCheck` needs to be invoked separately when there are more than one `FileCheck` for a single `ispc` output. Or there are multiple outputs that need to be checked.